src/os/ui/help/resetsettingsoption.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('os.ui.help.resetSettingsOption');
+
+goog.require('os.ui.EventType');
+goog.require('os.ui.help');
+goog.require('os.ui.util.ResetSettings');
+
+
+/**
+ * Add the Reset Settings option
+ */
+os.ui.help.resetSettingsOption.addToNav = function() {
+  var menu = os.ui.help.MENU;
+  menu.getRoot().addChild(os.ui.util.resetSettingsOptions);
```

Comment:
`os.ui.util.resetSettingsOptions` is only used to add this menu option, so I would suggest moving that object into this file.